parcels/field.py

```diff
@@ -309,8 +309,7 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
                 timeslices = []
                 dataFiles = []
                 for fname in data_filenames:
-                    with NetcdfFileBuffer(fname, dimensions, indices, netcdf_engine,
-                                          field_chunksize=False) as filebuffer:
+                    with NetcdfFileBuffer(fname, dimensions, indices, netcdf_engine, field_chunksize=False) as filebuffer:
```

Comment:
Specify here if `lock_file=False` or `lock_file=True`, to make choice explicit?

Reply:
we could make `lock_file=True` in this case (which is the default, which is why I did not specify that). Here, we need to lock because we actually access the time dimension of the NetCDF - and actual data access requires the lock.